distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java

```diff
@@ -271,11 +277,22 @@ public void beforeExam() throws Exception {
       loadResourceQueries(CSW_QUERY_RESOURCES, savedCswQueries);
       getServiceManager().startFeature(true, "spatial-wps");
       getServiceManager().startFeature(true, "sample-process");
+
     } catch (Exception e) {
       LoggingUtils.failWithThrowableStacktrace(e, "Failed to start required apps: ");
     }
   }
 
+  @Before
+  public void setup() throws Exception {
```

Comment:
I really don't like to be setting all these configurations unless we really have to